### PR TITLE
App review

### PR DESCRIPTION
**Pull request by bazelajoanna**
Add improvements and fix bugs in application.

### CODE REVIEW

File: public/locales/pl/translation.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "searchByTagIcon": "Ikona wyszukiwania po tagu",
     "filterByTagInput": "Input filtrujący po tagu"
   },
-  "navbar": {
+  "header": {
```
Comment (evulins) [inline public/locales/pl/translation.json:9]: ❤️

File: src/components/pages/ProblemsPage/index.tsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ const ProblemsPage: FC = () => {
           ))
         )}
       </Wrapper>
-      <ScrollToTopButton />
+      {problems.length != 0 && <ScrollToTopButton />}
```
Comment (evulins) [inline src/components/pages/ProblemsPage/index.tsx:67]: 👍

**Review by evulins — APPROVED** [review]
Great work!

**Review by franiakat — APPROVED** [review]
!

**Pull request merged**